code/game/objects/items/devices/beacon.dm

```diff
@@ -32,6 +32,21 @@
 	to_chat(user, "<span class='notice'>You [enabled ? "enable" : "disable"] the beacon.</span>")
 	return
 
+/obj/item/beacon/pickup(mob/living/user)
+	. = ..()
+	if(!iscarbon(user))
+		return FALSE
+	log_item(user, INVESTIGATE_VERB_PICKEDUP, TRUE)
+
+
+/obj/item/beacon/dropped(mob/living/user)
+	. = ..()
+	if(!iscarbon(user))
+		return FALSE
+	log_item(user, INVESTIGATE_VERB_DROPPED, TRUE)
```

Comment:
I could see this messaging admins becoming very annoying

Reply:
It's logging anyways. they should know everything about how things went.